src/main/java/bio/terra/service/dataset/flight/ingest/IngestCreateParquetFilesStep.java

```diff
@@ -85,6 +85,11 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
       ingestResponse.loadResult(fileLoadResults);
     }
 
+    // If loading from a payload, there is no path to report to the user
+    if (IngestUtils.isIngestFromPayload(context.getInputParameters())) {
```

Comment:
what would we be reporting to the user if we did not set this to null?

Reply:
We'd be reporting the path where we're temporarily landing the file that contains the data that the user has streamed in